src/log.c

```diff
@@ -337,7 +337,7 @@ static const char *strerror_result_string(const char *a, const char *b) { (void)
 
 void error_int( const char *prefix, const char *file, const char *function, const unsigned long line, const char *fmt, ... ) {
     // save a copy of errno - just in case this function generates a new error
-    int __errno = errno;
```

Comment:
Internally, OpenBSD defines an `__errno` function as part of the expansion for `errno`, so this name caused a compilation error.